### ISSUE TITLE
MY OPEN ORDERS ckickable for trading input

### ISSUE THREAD

**Issue by froooze** [bitshares/bitshares-ui#2699]
**Is your feature request related to a problem? Please describe.**
I get the price from the BUY/SELL order book to the input fields, by clicking. 
This does not work with the MY OPEN ORDERS 

**Describe the solution you'd like**
MY OPEN ORDERS should also be clickable like the BUY/SELL ORDERS.

**Comment by startailcoon** [comment]
Good point, been thinking the same thing recently.

**Comment by startailcoon** [comment]
# Task

- User clicks any of their rows in My Open Orders
- Price of row is used to fill in "Sell" and "Buy" for respectively in the same way order book does today.

**Comment by froooze** [comment]
Can I claim this?

**Comment by sschiessl-bcp** [comment]
> Can I claim this?

You can if you really want to :) Exchange code / entanglement is a mess IMO

**Comment by startailcoon** [comment]
This shouldn't be to difficult to create, but I agree that it's a tangled mess in that department. It's yours if you want @frooze

**Comment by froooze** [comment]
My idea was to use the code from BUY/SELL order click?

**Comment by startailcoon** [comment]
> 
> 
> My idea was to use the code from BUY/SELL order click?

Yes, that's the place to start. Use the callbacks used on the order book and apply to the my open orders component.

**Comment by startailcoon** [comment]
@froooze ping

**Comment by froooze** [comment]
@startailcoon: Is it ok, when I do a PR next week?
